.github/workflows/build.yml

```
@@ -104,7 +104,7 @@ jobs:
     - uses: actions/download-artifact@v3
     - name: Install dependencies
       run: |
-        apk --update add build-base bash gpgme-dev libarchive-tools libarchive-dev libtool doxygen libcrypto1.1
+        apk --update add build-base bash gpgme-dev libarchive-tools libarchive-dev libtool doxygen libcrypto1.1 xz
```

```suggestion
        apk --update add build-base bash gpgme-dev libarchive-tools libarchive-dev libtool doxygen libcrypto1.1
```

This one is not needed.